pandas/core/frame.py

```diff
@@ -5980,6 +5980,49 @@ def melt(self, id_vars=None, value_vars=None, var_name=None,
                     var_name=var_name, value_name=value_name,
                     col_level=col_level)
 
+    def explode(self, col_name, sep=None, dtype=None):
+        """
+        Create a new DataFrame where each element in each row
+        of a list-like column `col_name` is expanded to its own row
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        col_name : str
```

Comment:
I think we've been moving towards always using the full `column` instead of `col` for parameter names.

Reply:
how would we distinguish between the string name of the column and the column's data?

Reply:
I guess there's enough context here. I'll change to `column` for consistency then. But in general I'm still curious what the conclusion was for the question above.